build/ftest-base/pom.xml

```diff
@@ -60,6 +51,13 @@
         <type>pom</type>
         <scope>import</scope>
       </dependency>
+      <dependency>
+        <groupId>org.jboss.shrinkwrap.resolver</groupId>
+        <artifactId>shrinkwrap-resolver-depchain</artifactId>
+        <version>${version.shrinkwrap.resolver}</version>
+        <scope>test</scope>
+        <type>pom</type>
+      </dependency>
```

Comment:
The above dependency is already included in shrinkwrap-resolver-bom, so I think there is no need to add it here.